tools/azure-sdk-tools/devtools_testutils/envvariable_loader.py

```diff
@@ -96,7 +96,7 @@ def create_resource(self, name, **kwargs):
         load_dotenv(find_dotenv())
 
         if self.is_live:
-            self._set_mgmt_settings_real_values()
+            # self._set_mgmt_settings_real_values()
```

Comment:
Commenting this out for now--

But if we still need this functionality to be supported, we will need some sort of logic here to conditionally call that helper function (or to simply pass if not set and maybe raise an error)?